index.html

```diff
@@ -23,9 +24,15 @@
     <meta property="og:title" content="Virtual Assistant | Assistant Virtuel" />
     <meta property="og:description" content="Virtual Assistant | Assistant Virtuel" />
 
+    <!-- dcterms metadata -->
+    <meta name="dcterms.creator" content="ESDC_GC"/>
+    <meta name="dcterms.accessRights" content="2"/> 
+    <meta name="dcterms.service" content="ESDC-VA_MSCA-AV"/>
```

Comment:
Following Alpha, the content should probably be : ESDC-EDSC_VA-AV.

Reply:
I probably should've checked what the french version of ESDC was, my bad 😅